recipes/pyaerocom/meta.yaml

```diff
@@ -38,8 +38,6 @@ requirements:
     - pytest
 
 test:
-  requires:
-    -pytest
```

Comment:
This needed to have a space I think. YAML will interpret:
* `-pytest` as `"-pytest"` 
* `- pytest` as `["pytest"]`

Reply:
Ah, thanks a lot @chrisburr! I already suspected that it was related to that and removed it. Will update it and see if it works then. Cheers!